content/zh/blog/_posts/2018-08-29-kubernetes-testing-ci-automating-contributor-experience.md

```diff
@@ -0,0 +1,233 @@
+Today, the Kubernetes project spans over 125 repos across five orgs. There are 31 Special Interests Groups and 10 Working Groups coordinating development within the project. In the last year the project has had [participation from over 13,800 unique developers](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All) on GitHub.
+-->
+
+今天，Kubernetes项目跨越了5个组织125个仓库。有31个特殊利益集团和10个工作组在项目内协调发展。在过去的一年里，该项目有[来自13800多名独立开发人员的参与]（https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group？github上的orgid=1&var-period_name=last%20year&var metric=contributions&var-repogroup_name=all）。
+
+<!--
+On any given weekday our Prow instance [runs over 10,000 CI jobs](http://velodrome.k8s.io/dashboard/db/bigquery-metrics?panelId=10&fullscreen&orgId=1&from=now-6M&to=now); from March 2017 to March 2018 it ran 4.3 million jobs. Most of these jobs involve standing up an entire Kubernetes cluster, and exercising it using real world scenarios. They allow us to ensure all supported releases of Kubernetes work across cloud providers, container engines, and networking plugins. They make sure the latest releases of Kubernetes work with various optional features enabled, upgrade safely, meet performance requirements, and work across architectures.
+-->
+
+在任何给定的工作日，我们的Prow实例[运行超过10,000个CI工作]（http://velodrome.k8s.io/dashboard/db/bigquery-metrics?panelId=10&fullscreen&orgId=1&from=now-6M&to=now）; 从2017年3月到2018年3月，它有430万个工作岗位。 这些工作中的大多数涉及建立整个Kubernetes集群，并使用真实场景来实施它。 它们使我们能够确保所有受支持的Kubernetes版本跨云提供商，容器引擎和网络插件工作。 他们确保最新版本的Kubernetes能够启用各种可选功能，安全升级，满足性能要求，并跨架构工作。
+
+<!--
+With today’s [announcement from CNCF](https://www.cncf.io/announcement/2018/08/29/cncf-receives-9-million-cloud-credit-grant-from-google) – noting that Google Cloud has begun transferring ownership and management of the Kubernetes project’s cloud resources to CNCF community contributors, we are excited to embark on another journey. One that allows the project infrastructure to be owned and operated by the community of contributors, following the same open governance model that has worked for the rest of the project. Sound exciting to you? Come talk to us at #sig-testing on kubernetes.slack.com.
+-->
+
+今天[来自CNCF的公告]（https://www.cncf.io/announcement/2018/08/29/cncf-receives-9-million-cloud-credit-grant-from-google） - 注意到Google Cloud有开始将Kubernetes项目的云资源的所有权和管理权转让给CNCF社区贡献者，我们很高兴能够开始另一个旅程。 允许项目基础设施由贡献者社区拥有和运营，遵循对项目其余部分有效的相同开放治理模型。 听起来令人兴奋。 请来kubernetes.slack.com上的＃sig-testing与我们联系。
+
+<!--
+Want to find out more? Come check out these resources:
+-->
+
+想了解更多？ 快来看看这些资源：
+
+<!--
+* [Prow: Testing the way to Kubernetes Next](https://bentheelder.io/posts/prow)
+* [Automation and the Kubernetes Contributor Experience](https://www.youtube.com/watch?v=BsIC7gPkH5M)
+-->
+
+* [Prow：前往Kubernetes的方式]（https://bentheelder.io/posts/prow）
+* [自动化和Kubernetes贡献者体验]（https://www.youtube.com/watch?v=BsIC7gPkH5M）
```

Comment:
The link text is using the title of the linked page. I would consider leaving the original text.